src/ArtifactUploader.ts

```diff
@@ -28,7 +28,7 @@ export class GithubArtifactUploader implements ArtifactUploader {
     private async uploadArtifact(artifact: Artifact,
                                  releaseId: number,
                                  uploadUrl: string,
-                                 retry = 3) {
+                                 retry = 50) {
```

Comment:
I don't know that we'd want to default this to 50 for everyone, this could lead to some really long delays if there is a legit failure.